### PR TITLE
octopus: qa/tests: advanced octopus initial version to 15.2.10

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
https://pulpito.ceph.com/teuthology-2021-05-07_16:27:06-upgrade:octopus-p2p:octopus-p2p-stress-split-nautilus-distro-basic-smithi

https://pulpito.ceph.com/teuthology-2021-05-07_16:28:15-upgrade:octopus-p2p:octopus-p2p-parallel-nautilus-distro-basic-smithi

File: qa/suites/upgrade/octopus-p2p/octopus-p2p-stress-split/1-ceph-install/octopus.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 meta:
-- desc: install ceph/octopus v15.2.1
+- desc: install ceph/octopus v15.2.10
 tasks:
 - install:
-    tag: v15.2.1
+    tag: v15.2.10
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/suites/upgrade/octopus-p2p/octopus-p2p-stress-split/1-ceph-install/octopus.yaml:5]: changed to 15.2.11

File: qa/suites/upgrade/octopus-p2p/octopus-p2p-parallel/point-to-point-upgrade.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ meta:
    Run ceph on two nodes, using one of them as a client,
    with a separate client-only node.
    Use xfs beneath the osds.
-   install ceph/octopus v15.2.3 and the v15.2.x point versions
+   install ceph/octopus v15.2.10 and the v15.2.x point versions
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/suites/upgrade/octopus-p2p/octopus-p2p-parallel/point-to-point-upgrade.yaml:6]: yes

[the remark this answers is not in the document]
Comment (neha-ojha) [reply qa/suites/upgrade/octopus-p2p/octopus-p2p-parallel/point-to-point-upgrade.yaml:6]: sounds good, we should update the `desc:` for future reference :)

**Review by neha-ojha — APPROVED** [review]
nit: you can squash the commits into one, otherwise lgtm

**Pull request merged**